mkt/websites/views.py

```diff
@@ -48,3 +53,11 @@ def as_view(cls, **kwargs):
         # to be wrapped in transactions.
         view = super(WebsiteSearchView, cls).as_view(**kwargs)
         return non_atomic_requests(view)
+
+
+class ReviewersWebsiteSearchView(WebsiteSearchView):
+    permission_classes = [GroupPermission('Apps', 'Review')]
```

Comment:
We'll probably want a new permission like `Websites:Review`. Thoughts, @eviljeff ?